agent/agent-bootstrap/src/main/java/com/microsoft/applicationinsights/agent/bootstrap/diagnostics/status/StatusFile.java

```diff
@@ -116,9 +104,7 @@ public class StatusFile {
     VALUE_FINDERS.add(mf.getExtensionVersion());
 
     logDir = initLogDir();
-    directory = logDir + STATUS_FILE_DIRECTORY;
-    File dir = new File(logDir);
-    writable = dir.canWrite();
+    directory = DiagnosticsHelper.isOsWindows() ? logDir + "/Status" : logDir;
```

Comment:
it used to be lowercase `s`, even though case insensitive, can you check what case is used in App Services env (or do we create this directory ourselves)?

Reply:
it's case insensitive and we created it ourselves. use /Status because the whole path is using capital letter.. following the same naming convention.